common/src/main/java/org/dash/wallet/common/util/NavigationExtensions.kt

```diff
@@ -47,4 +48,13 @@ fun DialogFragment.dialogSafeNavigate(directions: NavDirections) {
             navController.navigate(directions)
         }
     }
+}
+
+enum class DeepLinkDestination(val deepLink: Uri) {
+    ReceiveDash(Uri.parse("android-app://hashengineering.darkcoin.wallet/payments/0")),
+    SendDash(Uri.parse("android-app://hashengineering.darkcoin.wallet/payments/1"))
+}
```

Comment:
Should `hashengineering.darkcoin.wallet` be replaced by a variable that is unique to the current app package id since we have a different id for testnet and dashpay and our devnet flavors?

Reply:
Not necessary, because the deep link is declared here and it stays the same across flavors: https://github.com/dashevo/dash-wallet/pull/1040/files#diff-9843f8fc1e09aaca34667187e3b60c8deb5858c04f4fd213d77e32b398ddcd92R45

So is confirmed by QA. We can, however, move it to a constant to avoid potential problems with refactoring later.